scripts/build/Jenkins/Jenkinsfile

```diff
@@ -854,7 +854,7 @@ finally {
                     "build_number": env.BUILD_NUMBER,
                     "repository_name": env.REPOSITORY_NAME,
                     "branch_name": env.BRANCH_NAME,
-                    "pipeline_name": GetRunningPipelineName(env.JOB_NAME),
+                    "pipeline_name": GetRunningPipelineName(env.JOB_NAME)[1],
```

Comment:
Since you're updating to just get the pipeline name instead you can just use `pipelineName` I think it's already stored in that variable.